dev/devicelab/manifest.yaml

```diff
@@ -68,6 +68,11 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["has-android-device"]
 
+  technical_debt__cost:
+    description: >
+      Estimates our technical debt (TODOs, analyzer ignores, etc).
+    stage: devicelab
```

Comment:
Looks like it won't work: https://github.com/flutter/cocoon/blob/master/commands/reserve_task.go#L118

Let's set it to "has-android-device" like the others in this section. I'll clean it up later.